src/server/components/lists/questionnaire.ts

```diff
@@ -59,11 +62,20 @@ export const questions: {
   },
   region: {
     getViewPartialName() {
-      return "questions/question-region.html";
+      return "questions/question-region.njk";
     },
     pageTitle(req) {
-      const { country } = getAllRequestParams(req);
-      return `Which area in ${startCase(country)} do you need a lawyer from?`;
+      const { country, serviceType } = getAllRequestParams(req);
+      const formattedCountry = startCase(country);
+
+      switch (serviceType) {
+        case ServiceType.covidTestProviders:
+          return `Where in ${formattedCountry} do you need to find a COVID-19 test provider?`;
+        case ServiceType.lawyers:
+          return `Which area in ${formattedCountry} do you need a lawyer from?`;
+        default:
+          return "";
```

Comment:
not a hill I would die (so happy for this to stay as it is) on but I find objects nicer when I don't need to do any fallthrough logic. 

```
const titles = {
  [ServiceType.covidTestProviders]: `Where in ${formattedCountry} do you need to find a COVID-19 test provider?`,
  [ServiceType.lawyers]: `Which area in ${formattedCountry} do you need a lawyer from?`
}
return titles[serviceType] ?? ""
```

minimal nesting ❤️